mealplanner-ui/src/pages/ShoppingList.tsx

```diff
@@ -64,13 +68,20 @@ export const ShoppingList = () => {
   interface Meal {
     id: string;
     name: string;
+    matchedProducts: Product[]
+  }
+
+  interface Product {
+    id: string;
+    productName: string;
+    price: any;
```

Comment:
Why the type cannot be number?

Reply:
Thats my bad, the type was 'any' in the relay file, so I used 'any' as well. I will change that